lib/rack/profiler.rb

```diff
@@ -14,8 +15,12 @@ class Profiler
                              'render_template.action_view',
                              'render_partial.action_view',
                              'process_action.action_controller',
+                             'endpoint_run.grape',
+                             'endpoint_render.grape',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.